meetings/2018-03-16-neuf-meeting.md

```diff
@@ -0,0 +1,165 @@
+* Dan: couple of individuals to sort of potentially reach out to invite, Myles as one of the leads of the modules WG and maybe someone
+  like Jordan Harbor who’s one of the participants.
+* Michael: Chris how closely aligned you think the people working on modules are to the kind of tooling issues and stuff
+  you were thinking, the sort of kind of things you want to address by this bringing this group together?
+* Chris: I’m not familiar with submodules group.
+* Michael: they’re focused on ES6 support in Node (Tierney: specifically, around the modules implementation),
+  how that ES6 support will be implemented. I guess the interaction could be, is that some of that may affect what 
+  some tooling can do and what some modules built on top can do. How do you test ES6 modules that might affect Mocha?
+* Chris: There’s some overlap, but I wouldn’t say necessarily any more overlap that any other.
+* Michael: Unless you think that’s a good fit I think we might want to be a bit cautious on. I’d like Chris to bring the group 
+  that he was thinking of together plus whoever else selects as being like in the tooling area and see what the concerns are,
+  and then figure out who to bring as opposed to trying to pre-figure that out.
+* Tierney: The thing I’d be most concerned about with pre-figuring that out is optimizing for the people who have already given feedback pretty extensively, like Myles and Jordan (involved in the project for quite a while), and I just be concerned that continuing to build like an echo chamber kind to ourselves and the feedback that we’ve already gotten, I’m elevating those voices over something with people who at least I’ve considered I had conversations with say feel like we’re kind of very much in echo chamber in a very exclusive environment, it’s hard to break into so I think we really to optimize for the ecosystem tooling people who haven’t really interacted with the node project.
+* Michael: I’ll like to have people come in tell us, what they’re thinking help us understand a bit better concerns 
+  and where they might want see things going and from that we can sort of figure out how do we bring additional people from the project
+  to align that versus some other way.
+* Tierney: I already reached out to a few people, I’ll point people in that direction.
+* Dan: My concern is: we are creating space for this discussion and then where is it go? Is not clear if is Comm-Comm, 
+  user feedback is not an incubation space for initiatives…  
+* Michael: I don’t think it need to go anywhere, our focus is to keep regular contacts with these groups and then once the contacts
+  takes place, if there’s concrete actions that need to happen out of those, like for example here’s a real pain point, 
+  our actions to figure out how to pull in the other people who can discuss that specific pain point have 
+  a specific meanings around that. You’re thinking is like do we need to go to a strategic initiative around this or a WG,
```

Comment:
The alternative is to spin up a team or strategic initiative.  (replace You're....or a WG)